content/en/building/guides/database/querying_training_card_telemetry.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "Querying Training Card Telemetry Data"
+linkTitle: "Querying Training Card Data"
+weight:
+description: >
+  How to use SQL queries to view metrics about Training Card usage
+relatedContent: >
+  building/guides/performance/telemetry/
+  core/overview/data-flows-for-analytics
+  building/guides/training/training-cards
+aliases:
+   - /apps/guides/database/querying_training_card_telemetry
+---
+
+Introduced in `4.2.0`, CHT has supported deployment of in-app training cards to facilitate remote training.
+
+Since interaction with training cards logs [telemetry data]({{< ref "building/guides/performance/telemetry" >}}), it is possible to view the data directly from CouchDB. However, it is more useful when you can run queries that provide useful metrics about the usage of training cards aggregated across many users, and interactions. With this in mind, it is typically easier to query the data using SQL from an [analytics database]({{< ref "core/overview/data-flows-for-analytics" >}}).
+
+This guide includes several SQL queries that can act as a starting point for identifying useful metrics. 
+
+In these examples the training cards set has an ID of `chp_self_assessment` which you can replace to get the metrics for any training card set.
+
+1. Total number of users who have viewed a particular training set:
+
+```
+SELECT
```

Comment:
I have not specifically worked with the `couchdb_users_meta` but I suspect the slowness is because your WHERE clauses use a JSON column without an index. If performance is critical you can use CHT Sync and write models that would pull out the relevant columns and normalise the table schema.

Reply:
Thanks Njuguna. For the purpose of this PR I will close it as it is... And explore CHT Sync later to compare.